### ISSUE TITLE
Smoke test failing on upstream devel branch of glusterfs

### ISSUE THREAD

**Issue by rkothiya** [gluster/project-infrastructure#108]
Smoke test failing for the following issue : 
https://github.com/gluster/glusterfs/pull/1786

```
13:53:58 ERROR: Exception(glusterfs-2020.12.08-0.08.2308.el7.src.rpm) Config(epel-6-i386) 0 minutes 7 seconds
13:53:58 INFO: Results and/or logs in: /home/jenkins/root/workspace/gh_strfmt_errors/RPMS/el6/i686/
13:53:58 INFO: Cleaning up build root ('cleanup_on_failure=True')
13:53:58 Start: clean chroot
13:54:02 Finish: clean chroot
13:54:02 ERROR: Command failed: 
13:54:02  # /usr/bin/yum --installroot /var/lib/mock/epel-6-i386/root/ -y --releasever 6 update --setopt=tsflags=nocontexts
13:54:02 
13:54:02 Build step 'Execute shell' marked build as failure
13:54:02 Archiving artifacts
13:54:02 Setting status of a2512d05e3ed519293109309e16e07285e387430 to FAILURE with url https://build.gluster.org/job/gh_strfmt_errors/556/ and message: 'FAIL - please fix for merge
13:54:02  '
13:54:02 Using context: Testing: strfmt errors in a 32 build
13:54:03 Finished: FAILURE
```

**Comment by deepshikhaaa** [comment]
It has been failing because of the unavailability of el6 mock config for further testing. Since we are also dropping EL6 support, we are testing it on different version. It is right now failing when the mock config used here is `fedora-33-i386`

**Comment by deepshikhaaa** [comment]
It has been fixed. Now we are using fedora-33-i386 mock config for this job. The issues associated or the follow-up issues have been resolved.

**Comment by mscherer** [comment]
However, I think 32 bits architectures are on the way out. Maybe we should find a way to verify that test that use cross compilation rather than using a whole 32 bits chroot ? Or see if there is gcc/clang warning that would have the same effect.

**Comment by amarts** [comment]
@mscherer I agree! For now, if we make sure we have no warnings in 32bit compilation, it is good for us. I would keep 99% usecases a priority than that 1%, mainly if its causing much delay in our reviews coming to standstill.

For now, till we figure this out, we can pause these tests and allow patches to go through. I volunteer our company to fix the issues seen in 32bit compilation as a separate patch once we figure out a better mechanism for this.

@xhernandez what do you say?

**Comment by mscherer** [comment]
In fact, I guess that we can also compile on a platform that will keep 32 bits for some time, like Debian. There was a article on that: https://lwn.net/Articles/838807/

**Comment by xhernandez** [comment]
> @mscherer I agree! For now, if we make sure we have no warnings in 32bit compilation, it is good for us. I would keep 99% usecases a priority than that 1%, mainly if its causing much delay in our reviews coming to standstill.
> 
> For now, till we figure this out, we can pause these tests and allow patches to go through. I volunteer our company to fix the issues seen in 32bit compilation as a separate patch once we figure out a better mechanism for this.
> 
> @xhernandez what do you say?

Sorry for the late answer. I missed the comment. Personally, I would like to make sure that 32 bits platforms at least compile.
